contrib/pkginstaller/Makefile

```diff
@@ -40,7 +40,6 @@ packagedir: podman_version package_root Distribution welcome.html
 	../../test/version/version > $(PACKAGE_DIR)/VERSION
 	echo -n $(ARCH) > $(PACKAGE_DIR)/ARCH
 	cp ../../LICENSE $(PACKAGE_DIR)/Resources/LICENSE.txt
-	cp hvf.entitlements $(PACKAGE_DIR)/
```

Comment:
wouldn't this still need to still copy the new file?

Reply:
yeah, I figured out how to test it, and this indeed was missing.